tests/driver_si70xx/main.c

```diff
@@ -44,25 +31,10 @@ int main(void)
 
     puts("SI7021 temperature and humidity sensor test application\n");
 
-    /* enable the sensor if test pin given */
-    if (TEST_PIN_EN != GPIO_UNDEF) {
```

Comment:
what's the solution for @basilfx devel board (which one are you using btw?) - because deleting this will make that fail, or not?